keyboard/android/src/main/java/com/capacitorjs/plugins/keyboard/Keyboard.java

```diff
@@ -132,10 +142,7 @@ private int computeUsableHeight() {
 
                 private boolean isOverlays() {
                     final Window window = activity.getWindow();
-                    return (
-                        (window.getDecorView().getSystemUiVisibility() & View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN) ==
-                        View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
-                    );
+                    return window.getDecorView().getFitsSystemWindows();
```

Comment:
This is always returning false, even if I overlay the status bar with status bar plugin, I think we can keep the old code for now and use a suppress warning in the funtion